src/home_robot/environment.yml

```diff
@@ -13,6 +13,7 @@ dependencies:
   - pinocchio
   - pytorch=2.0.0  # Compatibility with pytorch3d
   - pytorch-cuda=11.8
+  - cudatoolkit-dev
```

Comment:
@alexsax do you think this would help with our install issues?

Reply:
cc @yvsriram @ykarmesh for updates to install + instructions

Reply:
We should try installing from scratch once.

Reply:
@yvsriram best way to test a clean install is to just run the sim evaluation right? i can do this when i am in lab today

Reply:
Yes, that would be the best way. Thanks!

Reply:
@hello-binit i think we decided we need to remove this right?